### PR TITLE
SUBMARINE-585. [WEB] Edit existing experiments through UI

### PR DESCRIPTION
**Pull request by wang0630**
### What is this PR for?
Allow editing experiments through UI.

### What type of PR is it?
[Feature]

### Todos
* [ ] - Task

### What is the Jira issue?
[SUBMARINE-585](https://issues.apache.org/jira/browse/SUBMARINE-585)
### How should this be tested?
https://travis-ci.com/github/wang0630/submarine/builds/178169720

### Screenshots (if appropriate)
Aug-03-2020 02-02-44

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: submarine-workbench/workbench-web-ng/proxy.conf.js [hunk 1]
```diff
@@ -24,13 +24,13 @@ dotenv.config();
 const proxyConfig = [
   {
     context: ['/api'],
-    target: 'http://localhost:8080',
+    target: 'http://submarine:8080',
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/proxy.conf.js:27]:
```suggestion
    target: 'http://localhost:8080',
```

File: submarine-workbench/workbench-web-ng/proxy.conf.js [hunk 2]
```diff
@@ -24,13 +24,13 @@ dotenv.config();
...
     secure: false,
     changeOrigin: true
   },
   {
     context: '/ws',
-    target: 'ws://localhost:8080',
+    target: 'ws://submarine:8080',
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/proxy.conf.js:33]:
```suggestion
    target: 'ws://localhost:8080',
```

**Comment by pingsutw** [comment]
I suggest that we could change "Edit" to "Update"

**Comment by liuxunorg** [comment]
Will merge if no more comments.

**Pull request closed without merging**